R/tools.R

```diff
@@ -37,17 +76,22 @@ lint_dir <- function(path) {
 #' with the `legacy_max_lint_r_errors` option in `rhino.yml`.
 #' This can be useful when inheriting legacy code with multiple styling issues.
 #'
+#' @param paths Character vector of directories and files to lint.
+#' When `NULL` (the default), check `app` and `tests/testthat` directories.
+#'
 #' @return None. This function is called for side effects.
 #'
 #' @export
-lint_r <- function() {
+lint_r <- function(paths = NULL) {
+  if (is.null(paths)) {
+    paths <- c("app", "tests/testthat")
+  }
+  paths <- check_paths(paths)
```

Comment:
We won't need to return `paths` from this function now.
```suggestion
  check_paths(paths)
```